clang/test/SemaHLSL/ArrayTemporary.ll

```diff
@@ -0,0 +1,76 @@
+; ModuleID = '/Users/cbieneman/dev/llvm-project/clang/test/SemaHLSL/ArrayTemporary.hlsl'
```

Comment:
Is this `.ll` file necessary? It doesn't look like a lit test file, no `RUN` commands etc.